new_platforms/src/optee_common.c

```diff
@@ -7,17 +7,274 @@
 #include <openenclave/bits/report.h>
 #include <openenclave/bits/result.h>
 
+#include <mbedtls/x509_crt.h>
+
+#include <RiotStatus.h>
+#include <RiotEcc.h>
+#include <RiotCrypt.h>
+#include <TcpsId.h>
+
 #ifndef _In_
 #include "sal_unsup.h"
 #endif
 
+unsigned char tcpsOid[] = {0x67, 0x81, 0x05, 0x05, 0x04, 0x02};
+
+// Adopted from mbedtls
+oe_result_t
+GetClaimOidData(
+    uint8_t * DerExtensionBuffer,
```

Comment:
follow consistent style.  remove space between type and *.  Here and in line 36.  (And maybe other places, not sure.)

Reply:
Done.